src/libNode/DSBlockProcessing.cpp

```diff
@@ -647,6 +647,11 @@ bool Node::ProcessDSBlock(const vector<unsigned char>& message,
                 return false;
             }
 
+            if (BROADCAST_TREEBASED_CLUSTER_MODE)
```

Comment:
!LOOKUP_NODE_MODE needed, causing lookup to crash

Reply:
Lookup node won't reach that code statement. we already check !LOOKUP_NODE_MODE at line 550

Reply:
The comment is dislocated. Refers to VCblock and FInal block. Resolved